package.json

```diff
@@ -29,7 +29,7 @@
     "lit-html": "^1.1.2"
   },
   "devDependencies": {
-    "@internetarchive/ia-menu-slider": "0.1.0-alpha.1b6f614",
+    "@internetarchive/ia-menu-slider": "0.0.3-alpha.2",
```

Comment:
Why did this version decrease?

Reply:
i was going by this tag when i was cutting canary tags https://github.com/internetarchive/iaux-menu-slider/blob/master/package.json#L3